src/audio/module_adapter/module/cadence.c

```diff
@@ -632,11 +632,15 @@ cadence_codec_process(struct processing_module *mod,
 	}
 
 	API_CALL(cd, XA_API_CMD_EXECUTE, XA_CMD_TYPE_DO_EXECUTE, NULL, ret);
-	if (ret != LIB_NO_ERROR) {
+	//if (ret != LIB_NO_ERROR) {
```

Comment:
please, remove commented out code

Reply:
Will do, it was one of the formatting issues which made me make this a draft.